### PR TITLE
Feature AUT-3769 add `align-center` icon

### PR DESCRIPTION
**Pull request by wazelin**
# [AUT-3769](https://oat-sa.atlassian.net/browse/AUT-3769)

This PR adds the new `align-center` icon

image

## Dependencies
- [ ] https://github.com/oat-sa/tao-core-ui-fe/pull/599

[AUT-3769]: https://oat-sa.atlassian.net/browse/AUT-3769?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "@oat-sa/tao-core-libs": "^1.0.0",
         "@oat-sa/tao-core-sdk": "^3.1.1",
         "@oat-sa/tao-core-shared-libs": "^1.6.1",
-        "@oat-sa/tao-core-ui": "^3.5.0",
+        "@oat-sa/tao-core-ui": "github:oat-sa/tao-core-ui-fe#feature/AUT-3769/support-centered-image",
```
Comment (wazelin) [inline views/package.json:16]: - [ ] Upgrade to the stable version once released

File: views/css/auth-selector.css.map [hunk 2]
```diff
@@ -1 +1 @@
-{"version":3,"sourceRoot":"","sources":["file:///github/workspace/tao/views/scss/auth-selector.scss"],"names":[],"mappings":"AACE,oDACE,mBACA,2EACE,YAGJ,uDACE,mBACA,wEACE,mBAEF,yEACE","file":"auth-selector.css"}
\ No newline at end of file
+{"version":3,"sourceRoot":"","sources":["file:///Users/sergei.mikhailov/Sources/nextgen-stack/tao/tao/views/scss/auth-selector.scss"],"names":[],"mappings":"AACE,oDACE,mBACA,2EACE,YAGJ,uDACE,mBACA,wEACE,mBAEF,yEACE","file":"auth-selector.css"}
```
Comment (pribi) [inline views/css/auth-selector.css.map:1]: Local file is used here I think this is not the final one.

File: views/css/error-page.css.map [hunk 3]
```diff
@@ -1 +1 @@
-{"version":3,"sourceRoot":"","sources":["file:///github/workspace/tao/views/scss/error-page.scss"],"names":[],"mappings":"AAEA,YACE,aACA,uBACA,mBACA,sBACA,8BAEA,kBACE,aACA,gBACA,uCAEA,wBACE,YAGF,yBACE,mBACA,eACA,aACA,mBAGF,2BACE,eACA,gBAGF,8BACE,gBAGF,oBACE,YAIJ,cACE,YAIJ,QACE,WACA,sBACA,mBACA,YACA,yBACA,iBACA,eACA,eAEA,cACE,WACA,sBACA","file":"error-page.css"}
\ No newline at end of file
+{"version":3,"sourceRoot":"","sources":["file:///Users/sergei.mikhailov/Sources/nextgen-stack/tao/tao/views/scss/error-page.scss"],"names":[],"mappings":"AAEA,YACE,aACA,uBACA,mBACA,sBACA,8BAEA,kBACE,aACA,gBACA,uCAEA,wBACE,YAGF,yBACE,mBACA,eACA,aACA,mBAGF,2BACE,eACA,gBAGF,8BACE,gBAGF,oBACE,YAIJ,cACE,YAIJ,QACE,WACA,sBACA,mBACA,YACA,yBACA,iBACA,eACA,eAEA,cACE,WACA,sBACA","file":"error-page.css"}
```
Comment (pribi) [inline views/css/error-page.css.map:1]: Local file is used here I think this is not the final one.

File: views/css/auth-selector.css.map (hunk 2 above)
Comment (wazelin) [reply views/css/auth-selector.css.map:1]: That's perfectly fine.
The map files are re-generated on the extension-release step.

Comment (pribi) [reply views/css/auth-selector.css.map:1]: It's okay. If the map files are regenerated on the extension-release step, then why do we need to include these changes here in this PR?

Comment (wazelin) [reply views/css/auth-selector.css.map:1]: Otherwise, the reviewers would have to build the .css themselves to test them out.

**Review by ViktoriaKh-oat — APPROVED** [review]

**Review by olga-kulish — APPROVED** [review]

**Review by pribi — APPROVED** [review]

**Pull request merged**